sys-apps/systemd-utils/systemd-utils-254.5-r2.ebuild

```diff
@@ -425,14 +440,21 @@ multilib_src_install() {
 	if multilib_is_native_abi; then
 		if use boot; then
 			into /usr
-			dobin bootctl kernel-install
-			doman man/{bootctl.1,kernel-install.8}
-			# 90-loaderentry.install is generated from 90-loaderentry.install.in
-			exeinto usr/lib/kernel/install.d
-			doexe src/kernel-install/*.install
+			dobin bootctl
+			doman man/bootctl.1
 			insinto usr/lib/systemd/boot/efi
 			doins src/boot/efi/{linux$(efi_arch).{efi,elf}.stub,systemd-boot$(efi_arch).efi}
 		fi
+		if use kernel-install; then
+			dobin kernel-install
+			doman man/kernel-install.8
+			# This may be empty depending on whether we also enable sd-boot/ukify
+			keepdir usr/lib/kernel/install.d
+			if [[ -n "$(ls -A src/kernel-install)" ]]; then
```

Comment:
I would prefer a USE flag check instead of this `ls -A` hack.

Reply:
Yes this makes more sense indeed

Reply:
Actually, the systemd meson install adds some plugins already, if we simply install the same set of plugins by default in systemd-utils we avoid difference in behaviour between the two, and then we also don't need this whole condition after all.

Reply:
Sounds good.

Reply:
Fixed!